### PR TITLE
SquaredDifference function for TensorFlow RawOps  frontend  

### LINKED ISSUE #18994: SquaredDifference

### PR DESCRIPTION
**Pull request by Romiest**
Close #18994

### CODE REVIEW

**Comment by Romiest** [comment]
I have removed the changes made in the `.idea` folder

**Comment by kgmann** [comment]
Hey @Romiest, thanks for your contribution. Can you please address these
- IDE configuration files in the `.idea` folder are still present, please remove them 
- `int32` and `int64` are also part of the supported dtypes 
- The function `_squared_difference_helper` seems unnecessary, you can use `helpers.dtype_and_values` directly

**Comment by Romiest** [comment]
> Hey @Romiest, thanks for your contribution. Can you please address these
> 
> * IDE configuration files in the `.idea` folder are still present, please remove them
> * `int32` and `int64` are also part of the supported dtypes
> * The function `_squared_difference_helper` seems unnecessary, you can use `helpers.dtype_and_values` directly

Thanks for your response @kgmann ! I addressed everything you mentioned but I think there is something wrong with `_template__of_py_test.xml` as there are no differences between my file and the origin one. If there is anything else I should adjust or modify let me know 👍🏻

**Comment by kgmann** [comment]
ivy-gardener

**Comment by Romiest** [comment]
> ivy-gardener ✅ Ivy gardener has formatted your code. If changes are requested, don't forget to pull your fork.

Thanks, @kgmann , any other modifications?

**Comment by Romiest** [comment]
@kgmann

**Comment by Romiest** [comment]
Would appreciate if I can get a reply. Thank you. @kgmann

**Review by kgmann — CHANGES_REQUESTED** [review]
Hey @Romiest, sorry for the confusion in my previous comment. I made the review and actually forgot to submit it 😅

**Comment by kgmann** [comment]
ivy-gardener

File: ivy/functional/frontends/tensorflow/raw_ops.py [hunk 1]
```diff
@@ -507,6 +507,24 @@ def Square(*, x, name="Square"):
     return ivy.square(x)
 
 
+SquaredDifference = to_ivy_arrays_and_back(
+    with_supported_dtypes(
+        {
+            "2.12.0 and below": (
```
Comment (kgmann) [inline ivy/functional/frontends/tensorflow/raw_ops.py:513]: Please change the version to `2.13.0 and below` else the tests will fail.

**Review by kgmann — CHANGES_REQUESTED** [review]

**Comment by kgmann** [comment]
Hey @Romiest , the code looks good to me (apart from the tf version to be changed to `2.13.0`) but the test for the implemented function is failing for some backends. 
You can run it locally with this command
 `pytest ivy_tests/test_ivy/test_frontends/test_tensorflow/test_raw_ops.py::test_tensorflow_SquaredDifference`
Can you debug to find out why ? Please let me know how it's going. Thanks

**Comment by kgmann** [comment]
lgtm 👍🏿

**Pull request merged**